finishFeature/src/main/res/layout/finish_fragment.xml

```diff
@@ -0,0 +1,4 @@
+<androidx.compose.ui.platform.ComposeView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/finishFragment"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content" />
```

Comment:
you can replace this one with just without layout as [here](https://docs.google.com/presentation/d/1FloN_b3qwyYtIaWZy9sxFSi2-SJqDFGoqcC34wiPnSQ/edit#slide=id.gf5758fc5a5_0_56) in the fragment instead.